src/helpers/transactionPressHandler.ts

```diff
@@ -78,7 +86,22 @@ export const getMenuItems = (item: any) => {
     };
   }
 
+  const isReadOnly =
```

Comment:
i believe you could use canBeResubmitted instead, so we don't getState here and have the same behavior than speed ups and cancel tx

Reply:
No, `canBeResubmitted` is in my opinion done wrongly since it does not respect whether the wallet is read only or not. It is working based on the assumption if the tx is stored locally, it needs to be done on the same phone which implies that the wallet is imported as a "full" wallet. 
However, this is not always true, this is technically possible that someone made a tx, it failed, signed off (removed wallet) and then the wallet got to read only status by importing by the public key. 
I understand that this is less likely (but still possible!) while speeding up, canceling, because the tx is not pending forever, but this is not true for failure status

Reply:
@osdnk was right here, we chatted via slack